TransmitBias/scripts/AFI_IndividualAlignRawAndInitialMaps.sh

```diff
@@ -74,46 +101,73 @@ GlobalScripts="$HCPPIPEDIR/global/scripts"
 mkdir -p "$AFIFolder"/GradientDistortionUnwarp
 mkdir -p "$WorkingDirectory/xfms"
 
-wb_command -volume-reorient "$AFIin" RPI "$AFIFolder"/AFI_orig.nii.gz
-if [[ "$GradientDistortionCoeffs" == "" ]]
-then
-    cp "$AFIFolder"/AFI_orig.nii.gz "$AFIFolder"/AFI_orig_gdc.nii.gz
+if (( IsLongitudinal )); then 
+    AFI_files=(AFI_orig.nii.gz AFI_orig_gdc_warp.nii.gz AFI_orig_gdc.nii.gz)
+    for file in ${AFI_files[*]}; do 
+        if [ -f "$AFIFolderCross/$file" ]; then 
+            cp $AFIFolderCross/$file $AFIFolder/
+        fi
+    done
 else
-    "$GlobalScripts"/GradientDistortionUnwarp.sh --workingdir="$AFIFolder"/GradientDistortionUnwarp \
-        --coeffs="$GradientDistortionCoeffs" \
-        --in="$AFIFolder"/AFI_orig.nii.gz \
-        --out="$AFIFolder"/AFI_orig_gdc.nii.gz \
-        --owarp="$AFIFolder"/AFI_orig_gdc_warp.nii.gz
+    wb_command -volume-reorient "$AFIin" RPI "$AFIFolder"/AFI_orig.nii.gz
+    if [[ "$GradientDistortionCoeffs" == "" ]]
+    then
+        cp "$AFIFolder"/AFI_orig.nii.gz "$AFIFolder"/AFI_orig_gdc.nii.gz
+    else
+        "$GlobalScripts"/GradientDistortionUnwarp.sh --workingdir="$AFIFolder"/GradientDistortionUnwarp \
+            --coeffs="$GradientDistortionCoeffs" \
+            --in="$AFIFolder"/AFI_orig.nii.gz \
+            --out="$AFIFolder"/AFI_orig_gdc.nii.gz \
+            --owarp="$AFIFolder"/AFI_orig_gdc_warp.nii.gz
+    fi
 fi
 
 useAFI="$AFIFolder"/AFI_orig_gdc.nii.gz
 useT1w="$T1wFolder"/T1w_acpc_dc.nii.gz
 #use new bias field if PSN wasn't used
 if [[ "$ReceiveBias" != "" ]]
 then
-    tempfiles_create TransmitBias_biasresamp_XXXXXX.nii.gz biasresamp
-    wb_command -volume-resample "$ReceiveBias" "$AFIFolder"/AFI_orig_gdc.nii.gz TRILINEAR "$biasresamp"
-    #deal with 0s in receive field by pretending they were 1s
-    wb_command -volume-math 'AFI / (receive + (receive == 0))' "$AFIFolder"/AFI_orig_gdc_RC.nii.gz -fixnan 0 \
-        -var AFI "$AFIFolder"/AFI_orig_gdc.nii.gz \
-        -var receive "$biasresamp" -repeat
+    if (( !IsLongitudinal )); then 
```

Comment:
I think bash syntax requires a space here (otherwise it is interpreted as a history command).

```suggestion
    if (( ! IsLongitudinal )); then 
```